ocp4/profiles/coreos-ncp.profile

```diff
@@ -140,75 +133,75 @@ selections:
+    - sysctl_kernel_yama_ptrace_scope
+    - sysctl_kernel_perf_event_paranoid
+    - sysctl_user_max_user_namespaces
+    - sysctl_user_max_user_namespaces.role=unscored
+    - sysctl_user_max_user_namespaces.severity=info
+    - sysctl_kernel_unprivileged_bpf_disabled
+    - sysctl_net_core_bpf_jit_harden
 
     ## File System Settings
-    #- sysctl_fs_protected_hardlinks
-    #- sysctl_fs_protected_symlinks
+    - sysctl_fs_protected_hardlinks
+    - sysctl_fs_protected_symlinks
 
     ### Audit
-    #- var_auditd_flush=incremental_async
-    #- auditd_data_retention_flush
-    #- auditd_local_events
-    #- auditd_write_logs
-    #- auditd_log_format
-    #- auditd_freq
-    #- auditd_name_format
-    #- var_auditd_action_mail_acct=root
-    #- var_auditd_space_left_action=email
-    #- auditd_audispd_configure_remote_server
-    #- auditd_audispd_encrypt_sent_records
-    #- auditd_audispd_disk_full_action
-    #- auditd_audispd_network_failure_action
-
+    - service_auditd_enabled
+    - var_auditd_flush=incremental_async
+    - auditd_data_retention_flush
+    - auditd_local_events
+    - auditd_write_logs
+    - auditd_log_format
+    - auditd_freq
+    - auditd_name_format
+    - var_auditd_action_mail_acct=root
+    - var_auditd_space_left_action=email
+    - auditd_audispd_configure_remote_server
+    - auditd_audispd_encrypt_sent_records
+    - auditd_audispd_disk_full_action
+    - auditd_audispd_network_failure_action
 
     ### Module Blacklist
-    #- kernel_module_cramfs_disabled
-#    - kernel_module_bluetooth_disabled
-    #- kernel_module_sctp_disabled
-    #- kernel_module_firewire-core_disabled
-    #- kernel_module_atm_disabled
-    #- kernel_module_can_disabled
-    #- kernel_module_tipc_disabled
+    - kernel_module_cramfs_disabled
+    - kernel_module_bluetooth_disabled
+    - kernel_module_sctp_disabled
+    - kernel_module_firewire-core_disabled
+    - kernel_module_atm_disabled
+    - kernel_module_can_disabled
+    - kernel_module_tipc_disabled
 
     ### rpcbind
 
     ### Install Required Packages
     #- package_sssd-ipa_installed
-    #- package_aide_installed
-    #- package_firewalld_installed
-    #- package_iptables_installed
+    - package_aide_installed
+    - package_firewalld_installed
+    - package_iptables_installed
     #- package_libcap-ng-utils_installed
     #- package_openscap-scanner_installed
     #- package_policycoreutils_installed
     #- package_rng-tools_installed
-    #- package_sudo_installed
-    #- package_usbguard_installed
-    #- package_audispd-plugins_installed
+    - package_sudo_installed
+    - package_usbguard_installed
+    - package_audispd-plugins_installed
     #- package_scap-security-guide_installed
-    #- package_audit_installed
-    #- package_libreswan_installed
-    #- package_rsyslog_installed
-    #- package_rsyslog-gnutls_installed
+    - package_audit_installed
+    - package_libreswan_installed
+    - package_rsyslog_installed
```

Comment:
Why is rsyslog needed? Cluster-logging-operator uses fluentd.

Reply:
yes

Reply:
@jhrozek @JAORMX is lfuentd using FIPS libraries and encryptions and is it realtime off loading?

Reply:
Fluentd is what the cluster-logging-operator uses to send logs out. It's real time off loading. I'll get back to you on the FIPS part